src/Convert.hs

```diff
@@ -51,6 +51,7 @@ import qualified Convert.StringParam
 import qualified Convert.StringType
 import qualified Convert.Struct
 import qualified Convert.StructConst
+import qualified Convert.SystemTasks
```

Comment:
Minor: I think I'd prefer this conversion called "SeverityTask". There are plenty of system tasks that this conversion doesn't affect, so this more specific name may be clearer. I prefer singular conversion names, though there are existing inconsistencies.

Reply:
Actually, with all the changes I made today, I believe it is better to organize it as "SeverityTask". This change has been completed